src/containers/PersonContainer/CoverImage.tsx

```diff
@@ -36,17 +62,23 @@ const CoverImage: FC<CoverImageProps> = ({
   electoralArea,
 }: CoverImageProps) => {
   return (
-    <div>
-      <CoverImg
+    <Images>
+      <img
         className="mzp-c-card-image"
         src={
           seatPictureSrc ||
           EXAMPLE_COVER_VIEWS[Math.floor(Math.random() * EXAMPLE_COVER_VIEWS.length)]
         }
-        alt={`${electoralArea || "Default Image, no Elector Seat Image Available"}`}
+        alt={`${electoralArea || ""}`}
```

Comment:
do we have a default image stored somewhere?

Reply:
Yes, in /assets/images